src/orders/types.ts

```diff
@@ -18,27 +18,51 @@ type OrderFee = {
   basisPoints: string;
 };
 
+/**
+ * The latest OpenSea Order schema.
+ */
 export type OrderV2 = {
+  /** The date the order was created. */
   createdDate: string;
+  /** The date the order was closed. */
   closingDate: string | null;
+  /** The date the order was listed. Order can be created before the listing time. */
   listingTime: number;
+  /** The date the order expires. */
   expirationTime: number;
+  /** The hash of the order. */
   orderHash: string | null;
+  /** The account that created the order. */
   maker: OpenSeaAccount;
+  /** The account that filled the order. */
   taker: OpenSeaAccount | null;
+  /** The protocol data for the order. Only 'seaport' is currently supported. */
   protocolData: ProtocolData;
+  /** The contract address of the protocol. */
```

Comment:
these don't seem helpful - should we only add them on method definitions?

Reply:
<img width="857" alt="Screenshot 2023-07-28 at 3 43 17 PM" src="https://github.com/ProjectOpenSea/opensea-js/assets/14880835/b7e2b5c8-5158-4715-9bc6-57243e70fbb0">

You are probably right they are not necessary most of the time. I could have skipped a bunch of them. But users can still view the object without the details.